RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 -----
 - [*] Coupons: Fixed issue loading the coupon list from the local storage on initial load. [https://github.com/woocommerce/woocommerce-ios/pull/6463]
 - [*] Jetpack CP: Fixed crash when attempting to access WP-Admin with an invalid URL that has an unsupported scheme. [https://github.com/woocommerce/woocommerce-ios/pull/6502]
+- [internal] Loading ghost screens are refactored to avoid a potential crash. Please quickly smoke test them so they look OK. https://github.com/woocommerce/woocommerce-ios/issues/2448
```

Comment:
Super nit: we usually keep the link to the PR within square brackets 😅. I'd suggest updating the message to indicate the removal of duplicated code too:
```suggestion
- [internal] Loading screens are refactored to avoid duplicated code and a potential crash. Please quickly smoke test them to make sure that everything still works as before. [https://github.com/woocommerce/woocommerce-ios/issues/2448]
```

Reply:
Yeah, I changed the text as suggested and added the PRs related to these changes so they can have a hint of how to test it 👍